### PR TITLE
pysolr added

### PR DESCRIPTION
**Pull request by cehbrecht**
python library to work with apache solr.

### CODE REVIEW

File: recipes/pysolr/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    - requests >=2.9.1
+    - simplejson
+
+test:
+  imports:
+    - pysolr
+
+about:
+  home: https://github.com/django-haystack/pysolr/
+  license: BSD
```
Comment (synapticarbors) [inline recipes/pysolr/meta.yaml]: please add `license_file: LICENSE`

Comment (synapticarbors) [inline recipes/pysolr/meta.yaml]: change to: `license: BSD 3-clause`

File: recipes/pysolr/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+{% set version = "3.6.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: 9ccd0f8450d84a32fee9301aa97de506
```
Comment (synapticarbors) [inline recipes/pysolr/meta.yaml]: Please use the `sha256` hash

File: recipes/pysolr/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,38 @@
+{% set name = "pysolr" %}
+{% set version = "3.6.0" %}
```
Comment (synapticarbors) [inline recipes/pysolr/meta.yaml]: Strangely, the `__version__` in the package from pypi.io says 3.5.0 (although the tarball is labeled as 3.6.0)

Comment (cehbrecht) [inline recipes/pysolr/meta.yaml]: set `setup.py` has the correct version "3.6.0". The `__version__` was not adapted ... but i have looked in the github code and it was fixed after the 3.6.0 release:

https://github.com/django-haystack/pysolr/blob/master/pysolr.py#L61

**Pull request merged**